### ISSUE TITLE
what's CARLIN_length？

### ISSUE THREAD

**Issue by jefferyUstc** [ShouWenWang-Lab/snakemake_DARLIN#5]
Hi, DARLin Team, what’s “CARLIN_length” in the [tutorial](https://github.com/ShouWenWang-Lab/snakemake_DARLIN/blob/master/QC/single_cell_CARLIN-10x.ipynb)？Is this equal to `bc_length`?

<img width="918" alt="image" src="https://github.com/ShouWenWang-Lab/snakemake_DARLIN/assets/44384930/169d3cec-96cc-4d7f-923e-078d061ea970">

without the exact meaning of CARLIN_length, I‘m fully lost in the second part(Compare with CARLIN pipeline).

Thank you.

**Comment by ShouWenWang** [comment]
Yes, it is the total nucleotides of the DARLIN barcodes after deletion and insertion.

 ――
Shou-Wen Wang, PhD
Principal Investigator
School of Life Sciences | School of Sciences
Westlake University
Shilongshan ST #18, Xihu, Hangzhou, Zhejiang
https://www.shouwenwang-lab.com/


________________________________
From: jeffery ***@***.***>
Sent: Friday, January 5, 2024 4:25:14 PM
To: ShouWenWang-Lab/snakemake_DARLIN ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [ShouWenWang-Lab/snakemake_DARLIN] what's CARLIN_length？ (Issue #5)

You don't often get email from ***@***.*** Learn why this is important<https://aka.ms/LearnAboutSenderIdentification>

Hi, DARLin Team, what’s “CARLIN_length” in the tutorial<https://github.com/ShouWenWang-Lab/snakemake_DARLIN/blob/master/QC/single_cell_CARLIN-10x.ipynb>？Is this equal to bc_length?

image.png (view on web)<https://github.com/ShouWenWang-Lab/snakemake_DARLIN/assets/44384930/169d3cec-96cc-4d7f-923e-078d061ea970>

without the exact meaning of CARLIN_length, I‘m fully lost in the second part(Compare with CARLIN pipeline).

Thank you.

―

**Comment by Selecton98** [comment]
I changed the CARLIN_length into bc_length in the codes and it worked.

**Comment by ShouWenWang** [comment]
Cool!

We will update the tutorial.
